### PR TITLE
Old Telebaton icon+changelog

### PR DESCRIPTION
**Pull request by Cafaling**
aeiou

### CODE REVIEW

**Comment by Cafaling** [comment]
open
closed

File: tgstation.dme [hunk 1]
```diff
@@ -840,7 +840,6 @@
 #include "code\modules\admin\banjob.dm"
 #include "code\modules\admin\create_mob.dm"
 #include "code\modules\admin\create_object.dm"
-#include "code\modules\admin\create_poll.dm"
```
Comment (nalarmg) [inline tgstation.dme:843]: Uh?

**Comment by Cafaling** [comment]
Issue?

**Comment by nalarmg** [comment]
You're removing the create poll verb, why?

**Comment by Cafaling** [comment]
Didn't intend to. I'll figure out what went wrong.

**Pull request closed without merging**